roofit/roofitcore/src/RooProdPdf.cxx

```diff
@@ -827,7 +825,7 @@ Int_t RooProdPdf::getPartIntList(const RooArgSet* nset, const RooArgSet* iset, c
 //       cout<<"FK: Starting Composite Term"<<endl;
 
       RooArgSet compTermSet, compTermNorm;
-      RooFIter tIter = group->fwdIterator();
+      RooFIter tIter = group.fwdIterator();
```

Comment:
With not-a-RooLinkedList, this would be a range-based for loop.

Reply:
Okay! We can use `std::list<std::vector<RooArgSet*>>`